Source/ToolCore/NETTools/NETProjectSystem.cpp

```diff
@@ -421,6 +425,96 @@ namespace ToolCore
 
     }
 
+    /// handle the results of a recoompile, and auto play the project if successful
+    void NETProjectSystem::HandleNETBuildPlay(StringHash eventType, VariantMap& eventData)
+    {
+        using namespace NETBuildResult;
+        
+        if (eventData[P_SUCCESS].GetBool())
+        {
+            ATOMIC_LOGINFOF("NETBuild Success for project, now playing.");
+            VariantMap shortcutData;  // encode a shortcut even to send a play
+            shortcutData[UIShortcut::P_KEY] = KEY_P;
+            shortcutData[UIShortcut::P_QUALIFIERS] = QUAL_CTRL;
+            SendEvent(E_UISHORTCUT, shortcutData);
+        }
+        else
+        {
+            const String& errorText = eventData[P_ERRORTEXT].GetString();
+
+            ATOMIC_LOGERRORF("\n%s\n", errorText.CString());
+            ATOMIC_LOGERRORF("NETBuild Error for project, will not play.");
```

Comment:
> should be pretty easy
hah, I found the perfect vehicle with the E_EDITORMODAL event! Coded it up and ... there is no event handler for it. I'll add something.